packages/algorithm/lib/variants/bgoncharuck/algorithm.dart

```diff
@@ -50,27 +50,25 @@ class OfOldWayMagic implements MagicSpecialization {
     int userWillPower;
 
     /// will store a user polls arithmetic mean
-    Map<PollModelType, double> userPollsMean;
+    Map<PollModelType, double> userPollsMean = null;
```

Comment:
Yeah, I know that Dart says it is already null.
But I struggled with it so much that I want it to be here.
So I will be definitely sure it is null, even if it does nothing.